src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -57,6 +57,20 @@
     private $securityType;
 
     /**
+     * Update the timestamps (changed + created).
+     * This method is mapped to the PRE_PERSIST and PRE_UPDATE
+     * lifecycle events.
+     */
+    public function updateTimestamps()
```

Comment:
It could be a trait, but do we support traits? It could be in an abstract class -- but then what would hte scope be? SecurityBundle? SuluCore?